test/detection/utils/functions.py

```diff
@@ -0,0 +1,38 @@
+import random
+
+import numpy as np
+
+
+def generate_boxes(
```

Comment:
In `supervision/test/test_utils.py`, there are already two functions, `mock_detections` and `mock_key_points`. It seems we are duplicating some logic from `mock_detections`. Try to unify them. 

If that’s not possible, at least move `generate_boxes` to `supervision/test/test_utils.py` and make its arguments and naming conventions consistent with the existing functions. 

One quick improvement would be to replace the separate `W` and `H` arguments with a single `resolution_wh` argument, which we use throughout the codebase.